docs/introduction.md

```diff
@@ -62,6 +62,20 @@ You can share class data between running VMs, which can reduce the startup time
 ### Native data operations
 If your Java application manipulates native data, consider writing your application to take advantage of methods in the Data Access Accelerator API.
 
+The following functions are provided:
+
+- Arithmetic, copying, and shifting operations for packed decimal data.
+- Routines to convert between decimal data types stored in byte arrays and Java binary types.
```

Comment:
This bullet is a bit misleading; I read it as conversion between decimal types and Java binary types.  However, we also support conversion between different binary coded decimal types (i.e. Packed Decimal <-> External Decimal <-> Unicode Decimal).  

Perhaps we want to expand to say:
> Conversion operations between different binary coded decimal types and Java binary types.